### ISSUE TITLE
Unable to install with React 17

### ISSUE THREAD

**Issue by devgladstone** [wojtekmaj/react-pdf#690]
**Before you start - checklist**

 - [x] I followed instructions in documentation written for my React-PDF version
 - [x] I have checked if this bug is not already reported
 - [x] I have checked if an issue is not listed in [Known issues](https://github.com/wojtekmaj/react-pdf/wiki/Known-issues)
 - [x] If I have a problem with PDF rendering, I checked if my PDF renders properly in Mozilla Firefox

**Description**

I am using React 17 from Create-React-App and when trying to install react-pdf, I get the following error below. I think this package needs to allow for Version ^17.

```bash
npm i react-pdf
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: first-react@0.1.0
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   react@"^17.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.3.0" from react-pdf@5.0.0
npm ERR! node_modules/react-pdf
npm ERR!   react-pdf@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/derekgladstone/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/derekgladstone/.npm/_logs/2020-11-25T19_02_27_910Z-debug.log
```

**Steps to reproduce**

1. Create project with React 17.
2. Run `npm install react-pdf`
3. See error

**Expected behavior**

Successfully install

**Environment**

 - React-PDF version: 5.0.0
 - React version: 17.0.1

**Comment by wojtekmaj** [comment]
Try `v5.1.0-beta`. Compatibility with React 17 was *just* introduced.

**Comment by 787941287** [comment]
use yarn😊

**Comment by wojtekmaj** [comment]
Closing as available in 5.1.0-beta.
